### ISSUE TITLE
Publication Translations

### ISSUE THREAD

**Issue by tadean** [az-digital/az_quickstart#2440]
## Motivation
`az_multilingual` has recently been added as a module. Publications should be updated to reflect the fact that translations are available.

### Is your feature request related to a problem? Please describe.
Publications currently use only the default translation.

## Proposed Resolution
Publications should send the context of the current language to citeproc-php so the citation is translated where available.
